ramls/item.json

```
@@ -58,76 +60,97 @@
     },
 
     "itemLevelCallNumber": {
+      "description": "Level call number",
```

```suggestion
      "description": "Call number of the item",
```

The `itemLevel` part refers to this being for an item, rather than across holdings (I consider this redundant, because it is in the item record, however changing it would be a breaking compatibility change)